qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -1266,7 +1266,7 @@ def least_busy(
 
         candidates = []
         for backend in all_backends:
-            if backend["status"]["name"] == "online" and backend["status"]["reason"] == "available":
+            if backend["status"]["name"] == "online":
```

Comment:
Which API is this response coming from? This one? https://quantum.cloud.ibm.com/docs/en/api/qiskit-runtime-rest/tags/backends

Because I don't see `status` as being an object, but instead it's a string:

<img width="735" height="770" alt="image" src="https://github.com/user-attachments/assets/ed5f6c18-d412-4dd5-b7a8-4e71df297abb" />

Or is it `GET /backends` https://quantum.cloud.ibm.com/docs/en/api/qiskit-runtime-rest/tags/backends ?

<img width="730" height="777" alt="image" src="https://github.com/user-attachments/assets/40aba53e-8c42-4e3f-97ae-116ee9092253" />

I guess it must be the latter.